go/libkb/kbsig.go

```diff
@@ -329,6 +349,26 @@ func (arg ProofMetadata) ToJSON(m MetaContext) (ret *jsonw.Wrapper, err error) {
 	ret.SetKey("seqno", jsonw.NewInt64(int64(seqno)))
 	ret.SetKey("prev", prev)
 
+	// If this is a standard user link, arg.Me will be provided. It is not
+	// provided during a provisionee's KEX and for team sigs, where it is
+	// overridden. It is expected to be provided by PerUserKey and
+	// Stellar proofs as well.
+	var hPrevInfo HPrevInfo
+	if arg.Me != nil {
+		hPrevInfo = arg.Me.GetLastKnownHPrevInfo()
+	} else {
+		hPrevInfo = *arg.HPrevInfoOverride
```

Comment:
This is also kind of fragile. For example, if for some reason KEX changes to add more links than just the sibkey/subkey, this would be wrong and we would need to end up sending hPrevInfo pointers through KEX.